src/bokeh/core/json_encoder.py

```diff
@@ -158,37 +158,24 @@ def serialize_json(obj: Any | Serialized[Any], *, pretty: bool | None = None, in
     if pretty and indent is None:
         indent = 2
 
-    content: Any
-    buffers: list[Buffer]
-    if isinstance(obj, Serialized):
-        content = obj.content
-        buffers = obj.buffers or []
-    else:
-        content = obj
-        buffers = []
+    content = obj.content if isinstance(obj, Serialized) else obj
 
-    encoder = PayloadEncoder(buffers=buffers, indent=indent, separators=separators)
+    encoder = PayloadEncoder(indent=indent, separators=separators)
     return encoder.encode(content)
 
 #-----------------------------------------------------------------------------
 # Dev API
 #-----------------------------------------------------------------------------
 
 class PayloadEncoder(JSONEncoder):
-    def __init__(self, *, buffers: list[Buffer] = [], threshold: int = 100,
-            indent: int | None = None, separators: tuple[str, str] | None = None):
+    def __init__(self, *, indent: int | None = None, separators: tuple[str, str] | None = None):
         super().__init__(sort_keys=False, allow_nan=False, indent=indent, separators=separators)
-        self._buffers = {buf.id: buf for buf in buffers}
-        self._threshold = threshold
-
-    def default(self, obj: Any) -> Any:
-        if isinstance(obj, Buffer):
-            if obj.id in self._buffers: # TODO: and len(obj.data) > self._threshold:
-                return obj.ref
-            else:
-                return obj.to_base64()
+
+    def default(self, o: Any) -> Any:
```

Comment:
```suggestion
    def default(self, obj: Any) -> Any:
```

This could break people. Highly unlikely, but I don't think it is worth it.

Reply:
Change `obj` to `o` was suggested by linter, to match the definition from the base class. I can revert the rename, but I don't expect this to be an issue at all. Some would would have to extend this class and call super `default` with a keyword argument, which is very unlikely.

Reply:
I am OK with the change, there's no reason for users to be extending this or even interacting directly with it at all.

Reply:
Also fine with me only looked at the current diff.